docker-compose.yml

```
@@ -0,0 +1,23 @@
+version: '3.8'
+
+services:
+  rushdeal_postgres:
+    image: postgres:16-alpine
```

```suggestion
    image: postgres:18
```

16버전, alpine을 한 이유가 있을까요?

---

이전 프로젝트 진행할 때 찾아봤을 때는 alpine이 이미지가 가벼워서 일반 postgres 보다 빌드 배포같은 것도 빠르고 저장 공간도 아낄 수 있어서 로컬 개발 환경에 적합한데 제가 썼을때 16버전이 공식 안정버전이었습니다..!
변경하셔두 상관업서요 ㅇㅅㅇ!!

---

아 그러시면 16버전 하면 좋을 거 같습니다!
최신 버전이 있는데 따로 이유가 있으신지 확인하고 싶었습니다.